### ISSUE TITLE
[Documentation] tap list returns more than just installed packages

### ISSUE THREAD

**Issue by bryce-carson** [hwittenborn/tap#46]
**Steps to reproduce:**
1. `tap --help` and note the below text. https://github.com/hwittenborn/tap/blob/d8704743db7880aa36b7a05ee9e951c2ea7890b7/tap/cfg.py#L35
2. `tap list` and observe that uninstalled (but known) packages are listed.

**What I expected to happen:**
`tap list` lists only packages that are installed.

---

`tap list` should _only_ list installed packages, such as `apt list --installed` does.

**Comment by hiddeninthesand** [comment]
For what it's worth, `tap list -im` or longhand `tap list --installed --mpr-only` does this.

**Comment by bryce-carson** [comment]
Yep. Just clarifying the usage statement could be a small improvement.

Changing to "List packages" rather than "list installed packages" is another alternative, if the default is to list all packages.
